frontend/src/pages/modelRegistry/screens/ModelVersionDetails/ModelVersionDetailsView.tsx

```diff
@@ -51,22 +62,39 @@ const ModelVersionDetailsView: React.FC<ModelVersionDetailsViewProps> = ({
                 .then(refresh)
             }
           />
-          <EditableLabelsDescriptionListGroup
-            labels={getLabels(mv.customProperties)}
-            allExistingKeys={Object.keys(mv.customProperties)}
-            saveEditedLabels={(editedLabels) =>
-              apiState.api
-                .patchModelVersion(
-                  {},
-                  {
-                    customProperties: mergeUpdatedLabels(mv.customProperties, editedLabels),
-                  },
-                  mv.id,
-                )
-                .then(refresh)
-            }
-          />
+          {isArchiveVersion ? (
+            <DashboardDescriptionListGroup
+              title="Labels"
+              contentWhenEmpty="No labels"
+              isEmpty={getLabels(mv.customProperties).length === 0}
+            >
+              <LabelGroup data-testid="label-group">
+                {getLabels(mv.customProperties).map((label) => (
+                  <Label textMaxWidth="40ch" key={label} color="blue" data-testid="label">
+                    {label}
+                  </Label>
+                ))}
+              </LabelGroup>
+            </DashboardDescriptionListGroup>
+          ) : (
```

Comment:
You don't need this ternary / new block. If you add an `isEditable` prop to `EditableLabelsDescriptionListGroup`, you can pass its value down into `DashboardDescriptionListGroup` which already has an `isEditable` prop that is just only set to true currently. Then pass `isEditable={!isArchiveVersion}` to `EditableLabelsDescriptionListGroup` here.